diff --git a/gradle/globals.gradle b/gradle/globals.gradle
index 79a0c29384..cea779fb10 100644
--- a/gradle/globals.gradle
+++ b/gradle/globals.gradle
@@ -25,9 +25,6 @@ allprojects {
   // Repositories to fetch dependencies from.
   repositories {
     mavenCentral()
-    maven {         
-        url = uri("https://maven.searchscale.com/snapshots")
-    }
     mavenLocal()
     /* Reenable this if we need it again in future
     maven {
